assets/js/components/wp-dashboard/WPDashboardUniqueVisitors.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * WPDashboardUniqueVisitors component.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { STORE_NAME as MODULES_ANALYTICS } from '../../modules/analytics/datastore/constants';
+import { STORE_NAME as CORE_USER } from '../../googlesitekit/datastore/user/constants';
+import PreviewBlock from '../PreviewBlock';
+import ReportError from '../ReportError';
+import {
+	readableLargeNumber,
+	changeToPercent,
+} from '../../util';
+import DataBlock from '../data-block';
+import { parseTotalUsersData, userReportDataDefaults } from '../../modules/analytics/util';
+const { useSelect } = Data;
+
+const WPDashboardUniqueVisitors = () => {
+	const dateRange = useSelect( ( select ) => select( CORE_USER ).getDateRange() );
+
+	const args = {
+		dateRange,
+		...userReportDataDefaults,
+	};
```

Comment:
See above

Reply:
@kostyalmm See above, let's use `getDateRangeDates` to pass actual dates and specify all arguments manually instead of relying on the legacy function. It needs to be pretty much the same as https://github.com/google/site-kit-wp/blob/develop/assets/js/components/adminbar/AdminBarUniqueVisitors.js.